frontend/src/plugin/index.ts

```diff
@@ -0,0 +1,107 @@
+import helpers from '../helpers';
```

Comment:
Nice separation.  Would be nice to add a note to the top of each file explaining what is in the git commit message.

```ts
/**
 * The lib.ts file should carry the bits to be used by plugins whereas
 * the index.ts should have the functions that Headlamp itself needs for
 * loading the plugins.
*/

/**
 * The index.ts should have the functions that Headlamp itself needs for
 * loading the plugins.
 * The lib.ts file should carry the bits to be used by plugins whereas
*/

```